### DISCUSSION TITLE
Make the popup reusable for different entity

### DISCUSSION THREAD

**Discussion by BasicCPPDev** [Clooos/Bubble-Card#914] [Feature requests]
Adding a variables section so we can pass some entity to the card from the button press action would be a good idea i think.

Scenario : i have a popup to set the mode of room temperature to use. That popup card is same for all room. I can set mode to "auto", "manual", "boost", "off" via dedicated buttons.
I just need a couple of entities to send to the card to make it work for all my rooms.

Do you know any other way to achieve this using same popupcard?

**Comment by Clooos** [comment]
Hi! I've never tried this use case, but you can try to achieve that with streamline card, here is the link:

https://github.com/brunosabot/streamline-card
